src/main/java/org/checkerframework/specimin/SpeciminRunner.java

```diff
@@ -187,7 +190,20 @@ public static void performMinimization(
       if (isEmptyCompilationUnit(target.getValue())) {
         // target key will have this form: "path/of/package/ClassName.java"
         String classFullyQualfiedName = target.getKey().replace("/", ".");
-        classFullyQualfiedName = classFullyQualfiedName.replace(".java", "");
+        // this condition is actually always true
```

Comment:
Then why is it necessary to check it? Maybe we should have an else block that throws an exception if it somehow is false?